canonical-websites/www.ubuntu.com
# Update privacy notices copy

## Done

- Updated the copy of 6 privacy notices pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/dataprivacy](http://0.0.0.0:8001/legal/dataprivacy)
- Check that the newly inserted link goes to the correct place
- Check that the changes match the changes for these copydocs:

https://docs.google.com/document/d/1kQmhTCRSnx05oYEFcNr0Dq7UUFrznq-GvQlRxvNAZ9s/edit

https://docs.google.com/document/d/1Et7uuP82vYgk-sXTOeLLlKi0NzwCdDHkSlfLS7IClzo/edit

https://docs.google.com/document/d/1rEyCOZWoKKJRPIg5jhANfWVibZJ9QVfqj4MkeNIJp4M/edit

https://docs.google.com/document/d/1wMKqustrdcGK-yzGGe9we6nyHDvLUPukQaPpMA5T6wk/edit

https://docs.google.com/document/d/1ZSNnMiLJl6SGSV0D_cr6MJAwpkf6xzbKYJdTL9MxLHc/edit

https://docs.google.com/document/d/1lNercxxavqCCaFkLbQ-uHa9Mq6iZDMOSBDIUa48zDcU/edit


## Issue / Card

Fixes: #3691
